cluster/gce/config-default.sh

```diff
@@ -319,28 +320,27 @@ fi
 #   new subnetwork will be created for the cluster.
 ENABLE_IP_ALIASES=${KUBE_GCE_ENABLE_IP_ALIASES:-false}
 NODE_IPAM_MODE=${KUBE_GCE_NODE_IPAM_MODE:-RangeAllocator}
-if [ ${ENABLE_IP_ALIASES} = true ]; then
+if [ "${ENABLE_IP_ALIASES}" = true ]; then
```

Comment:
This script contains a mix of = and == for the string equals comparison. In bash the = operator is equivalent to ==, but my opinion is that being consistent and choosing one of them (personally I prefer the = since that is POSIX style) is preferred.

Reply:
That is a nice point.
But there is a lot of this inconsistency (= vs. ==) in other shellscript files and shellcheck command doesn't take care of that.
So it is nice to keep them for keeping the PR scope simple and we can take care of that after getting a consensus for doing that.

Reply:
That is a nice point too, but we also know that "take care of that after getting a consensus for doing that" is a euphemism for "never". :-)

Reply:
Thank you for checking this.
It is simpler that this PR changes only failures which shellcheck detects, I think. So, this 
 changes only the failures.
I've rebased this. If you are possible, could you review this? thanks.

Reply:
I have now added my comments!

Reply:
@joakimr-axis 
> That is a nice point too, but we also know that "take care of that after getting a consensus for doing that" is a euphemism for "never". :-)

Then why we need to apply point which seems difficult to get a consensus..